framework/tests/annotationclassloader/Makefile

```diff
@@ -23,33 +25,39 @@ FRAMEWORKBUILD := $(PROJECTDIR)/../../build/
 
 all: load-from-dir-test load-from-jar-test
 
-demo: demo1 demo2
-
-# loads from framework.jar
+# ======================================================
+# demo or manual test usage:
+# loads from build directories
 demo1:
+	@echo "***** This command is expected to produce an error on line 7:"
 	$(JAVAC) \
-	-J-Xbootclasspath/p:$(FRAMEWORKJAR):$(JAVACJAR):$(PROJECTDIR) \
+	-J-Xbootclasspath/p:$(DATAFLOWBUILD):$(JAVACUTILBUILD):$(STUBPARSERBUILD):$(FRAMEWORKBUILD):$(JAVACJAR):$(PROJECTDIR) \
 	-processor org.checkerframework.common.aliasing.AliasingChecker \
 	-Anomsgtext \
 	-AprintErrorStack \
 	-Astubs=$(PROJECTDIR)/../aliasing/stubfile.astub \
 	LoaderTest.java
 
-# loads from framework/build
+# loads from framework.jar
 demo2:
+	@echo "***** This command is expected to produce an error on line 7:"
 	$(JAVAC) \
-	-J-Xbootclasspath/p:$(DATAFLOWBUILD):$(JAVACUTILBUILD):$(STUBPARSERBUILD):$(FRAMEWORKBUILD):$(JAVACJAR):$(PROJECTDIR) \
+	-J-Xbootclasspath/p:$(FRAMEWORKJAR):$(JAVACJAR):$(PROJECTDIR) \
 	-processor org.checkerframework.common.aliasing.AliasingChecker \
 	-Anomsgtext \
 	-AprintErrorStack \
 	-Astubs=$(PROJECTDIR)/../aliasing/stubfile.astub \
 	LoaderTest.java
 
+# ======================================================
+# ant test usage
+# loads from build directories
 load-from-dir-test:
 	-$(JAVAC) -J-Xbootclasspath/p:$(DATAFLOWBUILD):$(JAVACUTILBUILD):$(STUBPARSERBUILD):$(FRAMEWORKBUILD):$(JAVACJAR):$(PROJECTDIR) -processor org.checkerframework.common.aliasing.AliasingChecker -Anomsgtext -AprintErrorStack -Astubs=$(PROJECTDIR)/../aliasing/stubfile.astub LoaderTest.java > Out.txt 2>&1
 	diff -u Out.txt Expected.txt
```

Comment:
I would also break these long command-lines to make the executions easier to read.